### PR TITLE
OIDC Support

### PR DESCRIPTION
**Pull request by haritzsaiz**
Removing keycloak-dependant component for authenticating users. Refactored code to use more generic authentication approach using OIDC standard.

Some AWS Cognito specific implementation was done to adjust the logout experience since cognito doesn't implement the `end_session_endpoint` as specified by the standard

### CODE REVIEW

File: public/env-config.js
Comment (jjrodrig) [inline public/env-config.js:1]: Could this file be replaced by a blank config in stead of a local development configuration?

We should avoid to push internal configs into the main repo.

File: src/App.tsx [hunk 1]
```diff
@@ -1,27 +1,36 @@
-import React from "react";
-import { ReactKeycloakProvider } from "@react-keycloak/web";
-
 import { Provider } from "react-redux";
 import { store } from "ducks";
-import keycloak from "./keycloak";
 import { DashboardLayout } from "views/DashboardLayout";
+import { AuthProvider } from "react-oidc-context";
+import * as oidc from "oidc-client-ts";
+// console.log(oidc.Version);
```
Comment (jjrodrig) [inline src/App.tsx:6]: Please remove commented logging code

File: src/views/AlertsView/SubscribeDialog.tsx [hunk 2]
```diff
@@ -76,10 +76,12 @@ export const SubscribeDialog: React.FC<Props> = ({ event, isOpen, onClose }) =>
 
     useEffect(() => {
         const init = async () => {
-            const profile = await keycloak.keycloak.loadUserProfile();
-            setEmail(profile.email);
-            if (selectedChannelType === "email") {
-                setSelectedSubscriptionConfig({ email_address: profile.email });
+            console.log(auth.user);
```
Comment (jjrodrig) [inline src/views/AlertsView/SubscribeDialog.tsx:79]: Is this log useful?

**Review by jjrodrig — CHANGES_REQUESTED** [review]
Thanks for the PR.
In general is Ok but please remove unnecesary logging, comments and internal development configs.

**Comment by haritzsaiz** [comment]
I've updated the pr with the comments you mentioned

File: src/ducks/services/api/token.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { IdTokenClaims, User } from "oidc-client-ts";
 
 export const getUser = (): User | null => {
     console.log("b");
```
Comment (jjrodrig) [inline src/ducks/services/api/token.ts:4]: This log seems not to be useful, can we remove it?

**Review by jjrodrig — CHANGES_REQUESTED** [review]

**Review by jjrodrig — APPROVED** [review]

**Pull request merged**